spec/big_files/big_files_spec.rb

```diff
@@ -71,10 +78,32 @@ def expect_source_globber_used(glob, exclude_glob)
 
         it 'runs' do
           expect_source_globber_used(glob, exclude_glob)
-          file_1 = expect_file_processed('file_1', 1)
-          file_2 = expect_file_processed('file_2', 2)
+          file_1 = expect_file_processed('file_1', 4)
+          file_2 = expect_file_processed('file_2', 3)
+          file_3 = expect_file_processed('file_3', 2)
+          file_4 = expect_file_processed('file_4', 1)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.